model/src/pyrenew/latent/infectionswithfeedback.py

```diff
@@ -37,6 +37,16 @@ class InfectionsWithFeedback(RandomVariable):
     This class computes infections, given Rt, initial infections, and generation
     interval.
 
+    Parameters
+    ----------
+    infection_feedback_strength : RandomVariable
+        Infection feedback strength.
+    infection_feedback_pmf : RandomVariable
+        Infection feedback pmf.
+    infections_mean_varname : str, optional
```

Comment:
Making a note that these varnames should not have the word `mean` in them, but that's a separate PR.